### ISSUE TITLE
Negative temperature values are being read as 327 degrees Celsius

### ISSUE THREAD

**Issue by ixevix** [kardoo/ruuvitag-scanner-scala#1]
I'm not that good at scala but can you check if the values are being saved correctly? I had a setup with Ruuvicollector and it collected the values properly https://github.com/Scrin/RuuviCollector

![grafana_values](https://user-images.githubusercontent.com/820718/146517456-4525af37-7040-4b1e-bdbc-a6038c953c64.png)

**Comment by ixevix** [comment]
Using this scala version on archlinux since the bluetooth tools hcidump etc are not available anymore so this is the only working Ruuvitag scanner.

**Comment by kardoo** [comment]
Will the python version suit you?
I will not have time to debug it (nor negative temperatures)

**Comment by ixevix** [comment]
Which one is the python version?

**Comment by ixevix** [comment]
Actually I might be able to pull it off using debian and docker

**Comment by ixevix** [comment]
Yeah using this works as a workaround https://github.com/Scrin/RuuviCollector/blob/master/Dockerfile

**Comment by kardoo** [comment]
If needed I can commit a python version.

**Comment by ixevix** [comment]
I don't need it now, thanks.
